### PR TITLE
Improve documentation of builders.

### PR DESCRIPTION
**Pull request by migmedia**

### CODE REVIEW

File: src/generator/event_builder.rs [hunk 1]
```diff
@@ -71,6 +88,7 @@ impl DtStart {
         DtEnd(self.0)
     }
 
+    /// Start of a multi-hole-day event.
```
Comment (dcormier) [inline src/generator/event_builder.rs:91]:
```suggestion
    /// Start of a multi-whole-day event.
```

File: src/generator/event_builder.rs [hunk 2]
```diff
@@ -80,6 +98,7 @@ impl DtStart {
         DtEndDate(self.0)
     }
 
+    /// A one-day-event.
```
Comment (dcormier) [inline src/generator/event_builder.rs:101]:
```suggestion
    /// A one-day event.
```

File: src/generator/event_builder.rs [hunk 3]
```diff
@@ -91,9 +110,9 @@ impl DtStart {
 }
 
 impl DtEndDate {
-    /// Sets the `DTEND` of the event. Since a event from 9:00 - 10:00 has stoppt
-    /// at 10 and a new one can start. The `end_day` has to be the next day. This
-    /// `value` is **not inclusive**.
+    /// Sets the `DTEND` of the event. The `end_day` has to be the next day. This
+    /// `value` is **not inclusive**. Analog an event from 9:00 - 10:00 has stoppt
```
Comment (dcormier) [inline src/generator/event_builder.rs:114]:
```suggestion
    /// `value` is **not inclusive**. Analog an event from 9:00 - 10:00 has stopped
```

**Comment by migmedia** [comment]
@dcormier thank you for correcting my typos.

**Comment by Peltoche** [comment]
Hello @migmedia and @dcormier, thanks for this PR

**Review by Peltoche — APPROVED** [review]

**Pull request merged**